obspy/io/nordic/core.py

```diff
@@ -457,7 +459,13 @@ def _read_uncertainty(tagged_lines, event):
     except ValueError:
         pass
     if errors['x_err']:
-        warnings.warn("Hypocentral Error estimates found, but not converted.")
+        e = _ellipse.from_uncerts(errors['x_err'],
+                                  errors['y_err'], errors['xy_cov'])
+        event.origins[0].origin_uncertainty = OriginUncertainty(
```

Comment:
I have a comment in the source that there shouldn't be more than one error line, but I don't know where that assumption came from.  At the moment, if more than one origin is in the s-file then only the first one is able to be associated with an uncertainty, and that uncertainty isn't necessarily the correct uncertainty (origin 0 could not have any uncertainty with it, but origin 1 might do, then origin 1's uncertainty would be incorrectly linked to origin 0).  If you have any suggestions on this that would be great. Otherwise I can have a think as well. Sorry for the state of the code - it's not very nice and could do with a refactor.

Reply:
The SEISAN manual does state that there is only one 'E' line in an "S-file", which applies to the "prime" epicenter.